iPetCare/API/Controllers/SpeciesController.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using API.Security;
+using Application.Dtos.Species;
+using Application.Dtos.Users;
+using Application.Interfaces;
+using Domain.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers
+{
+    [Route("api/species")]
+    [ApiController]
+    public class SpeciesController : Controller
+    {
+        private readonly ISpeciesService _speciesService;
+
+        public SpeciesController(ISpeciesService speciesService)
+        {
+            _speciesService = speciesService;
+        }
+
+        [Authorize(Roles = Role.Administrator)]
+        [HttpPost]
+        public async Task<ActionResult<SpeciesCreateDtoResponse>> Create(SpeciesCreateDtoRequest dto)
```

Comment:
Jeśli poniżej piszemy GetSpecies to moglibyśmy od razu nazwać CreateSpecies. W dzialaniu oczywiście żadna różnica ale jest bardziej jednakowo